letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -385,9 +393,9 @@ UNLIKELY_EOF
     # ---------------------------------------------------------------------------
     DeterminePythonVersion
     if ! REMOTE_VERSION=`"$LE_PYTHON" "$TEMP_DIR/fetch.py" --latest-version` ; then
-      echo "WARNING: unable to check for updates."
+      say "WARNING: unable to check for updates."
```

Comment:
I think this should be `error` as well so the user is notified `certbot-auto` encountered a problem.